spec/integration/system_pager_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+RSpec.describe TTY::Pager::SystemPager do
+  around :each do |example|
+    Dir.mktmpdir do |dir|
+      FileUtils.cd(dir) do
+        example.run
+      end
+    end
+  end
+
+  it "paginates some text" do
+    output_path = Pathname.new("output.txt")
+    described_class.page("Some text", command: "ruby #{fixtures_path("cat.rb")} > #{output_path}")
+
+    expect(output_path.read).to eq "Some text"
+  end
+
+  it "allows pagination to happen asynchronously" do
+    output_path = Pathname.new("output.txt")
+    pager_command = "ruby #{fixtures_path("external_pager.rb")} #{output_path}"
```

Comment:
I wonder what is the benefit of using `external_pager.rb` over the `cat.rb` here? It seems that the aim is to test the block form writing and the `cat.rb` should do the job, e.i. pipe 'writes' to `output.txt`. I would probably consider replacing all integration tests to use `cat.rb` instead without losing the 'essence' of what is being tested.

Reply:
The reason is I can't get it to work locally if I print to stdout instead of to a file. Doing `.read` waits for all the content before returning it, so the kill won't do what it's supposed to. And rewriting the `external_pager.rb` script as a cat is giving me trouble. I'll try your suggestion.